torchvision/prototype/transforms/functional/_color.py

```diff
@@ -312,7 +312,13 @@ def posterize(inpt: features.InputTypeJIT, bits: int) -> features.InputTypeJIT:
         return posterize_image_pil(inpt, bits=bits)
 
 
-solarize_image_tensor = _FT.solarize
+def solarize_image_tensor(image: torch.Tensor, threshold: float) -> torch.Tensor:
+    if threshold > _FT._max_value(image.dtype):
```

Comment:
As discussed offline with @pmeier, we will revert the use of `_max_value` here temporarily and hardcode back the 255 value. This change should happen across V1 and V2 and it's a bug fix. But this should happen on a separate PR that changes it from everywhere to avoid issues.